qa/README.md

```diff
@@ -5,6 +5,16 @@ Every pull request to the bitcoin repository is built and run through
 the regression test suite. You can also run all or only individual
 tests locally.
 
+Test dependencies
+=================
+Before running the tests, the following must be installed.
+
+Unix
```

Comment:
Looks like the command here doesn't even apply to all GNU/Linux based systems but only to apt-get based systems. Unix -> Linux at least?